File: CS/DXApplication206/CustomGridView.cs
using DevExpress.XtraGrid;

namespace DXApplication206 {
    public class CustomGridView : DevExpress.XtraGrid.Views.Grid.GridView {
        public CustomGridView() {
        }

        public CustomGridView(GridControl grid) : base(grid) {
        }

        protected override string ViewName {
            get {
                return "CustomGridView";
            }
        }
    }
}

File: VB/DXApplication206/CustomGridView.vb
Imports DevExpress.XtraGrid

Namespace DXApplication206
    Public Class CustomGridView
        Inherits DevExpress.XtraGrid.Views.Grid.GridView

        Public Sub New()
        End Sub

        Public Sub New(ByVal grid As GridControl)
            MyBase.New(grid)
        End Sub

        Protected Overrides ReadOnly Property ViewName() As String
            Get
                Return "CustomGridView"
            End Get
        End Property
    End Class
End Namespace